bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/GoogleCloudResourcePrefixInterceptor.java

```diff
@@ -31,6 +31,9 @@ public class GoogleCloudResourcePrefixInterceptor extends HeaderInterceptor {
   public static final Metadata.Key<String> GRPC_RESOURCE_PREFIX_KEY =
       Metadata.Key.of("google-cloud-resource-prefix", Metadata.ASCII_STRING_MARSHALLER);
 
+  public static final Metadata.Key<String> GRPC_PARAMS_KEY =
```

Comment:
You can add a comment for the javadoc here.

Reply:
I dont want this to be part of public docs. Its an internal implementation detail. I'll add a code comment tho

Reply:
SG